jena-arq/src/main/java/org/apache/jena/sparql/engine/http/HttpQuery.java

```diff
@@ -298,6 +292,19 @@ public InputStream exec() throws QueryExceptionHTTP {
             throw jEx;
         }
     }
+    
+    private void contextualizeCompressionSettings() {
+        final RequestConfig.Builder builder = RequestConfig.copy(context.getRequestConfig());
+        builder.setContentCompressionEnabled(allowGZip || allowDeflate);
```

Comment:
Is it is the case that the two settings are no longer independent? If so why bother maintaining  independent settings?